Description: Disable unavailable dep quanta
 which for the most part adds the ability to use Time Stamp Counter (TSC) over
 other timing libraries, which is sometimes useful but almost always hard to
 get right. It also offers a time mocking ability, which is not used here.
Last-Update: 2025-08-16
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -190 +190 @@
-[dependencies.quanta]
+[disabled.dependencies.quanta]
@@ -257 +257 @@
-quanta = ["dep:quanta"]
+#quanta = ["dep:quanta"]
